magefiles/mage.go

```diff
@@ -539,10 +539,22 @@ func (Test) Hardening(ctx context.Context) error {
 	)
 }
 
+// Run QA tests (arguments: {testGroup} {testPattern})
+func (Test) QA(ctx context.Context, testGroup, testPattern string) error {
+	mg.Deps(mg.F(buildPackageDocker, "deb", "-cover"))
+	return qa(ctx, testGroup, testPattern)
+}
+
+// Run QA tests (arguments: {testGroup} {testPattern})
+func (Test) QASnap(ctx context.Context, testGroup, testPattern string) error {
```

Comment:
Added `mage` target to execute QA tests on SNAP package. 
It should be executed under `systemd` environment - Snap package can be installed only in that environment. One choice is to use Ubuntu VM and execute tests there to not interfere work on host.
Did some adjustments in Python code - but not all tests are adopted.